Gemfile.lock

```diff
@@ -72,6 +72,13 @@ GEM
     docile (1.4.0)
     erubi (1.10.0)
     execjs (2.8.1)
+    factory_bot (6.2.1)
```

Comment:
Interested to see how this works. Did you use the Gist from our "What's a Dev?"

Reply:
Interested in this as well